### PR TITLE
fix the crash in distillation if loss_weights set to be [1, 0] in yam…

### PR DESCRIPTION
**Pull request by sywangyi**
…l config

Signed-off-by: Wang, Yi A <yi.a.wang@intel.com>

# What does this PR do?

Fixes # (issue)
fix the crash in distillation if loss_weights set to be [1, 0] in yaml config

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sywangyi** [comment]
@jiqing-feng @echarlaix please notice this issue

**Comment by echarlaix** [comment]
Good catch @sywangyi, thanks for fixing this

File: optimum/intel/neural_compressor/trainer.py [hunk 1]
```diff
@@ -562,12 +562,12 @@ def compute_loss(self, model, inputs, return_outputs=False):
                 teacher_outputs = self.agent.criterion.teacher_model_forward(
                     inputs, teacher_model=self.agent.teacher_model._model
                 )
-
-            teacher_logits = self._get_logits(teacher_outputs)
-            distillation_loss = self.compute_distillation_loss(logits, teacher_logits)
-            loss *= self.agent.criterion.loss_weights[0]
-            loss += distillation_loss * self.agent.criterion.loss_weights[1]
-            loss /= self.agent.criterion.loss_weights[0] + self.agent.criterion.loss_weights[1]
+            if teacher_outputs is not None:
```
Comment (echarlaix) [inline optimum/intel/neural_compressor/trainer.py:565]: Why not adding additional constraint before applying distillation instead such as ` self.agent.criterion.loss_weights[1] > 0`.

Comment (echarlaix) [reply optimum/intel/neural_compressor/trainer.py:565]: Also `self.agent.criterion.loss_weights[1] + self.agent.criterion.loss_weights[0] > 0`  should be checked

**Pull request merged**